google/cloud/bigtable/tests/filters_integration_test.cc

```diff
@@ -306,11 +306,13 @@ TEST_F(FilterIntegrationTest, CellsRowLimit) {
     auto inserted = actual.emplace(cell.row_key(), 0);
     inserted.first->second++;
   }
-  std::map<bigtable::RowKeyType, int> expected{{prefix + "/one-cell", 1},
-                                               {prefix + "/two-cells", 2},
-                                               {prefix + "/many", 3},
-                                               {prefix + "/many-columns", 3},
-                                               {prefix + "/complex", 3}};
+  using bigtable::RowKeyType;
```

Comment:
If we're going to use this to shorten the initializers (which I'm ambivalent on), should we also use it for the other instances of RowKeyType in this function?

Reply:
On second thought (thanks for asking) the longer name is fine. Fixed.